packages/api/controllers/nomination.js

```diff
@@ -51,4 +51,25 @@ const createNomination = async (req, res) => {
   }
 };
 
-module.exports = { getNominationById, findAllNominataions, createNomination };
+const updateNomination = async (req, res) => {
+  const { id } = req.params;
+  try {
+    const nomination = await db.Nomination.update(
+      { status: req.body.status },
+      { where: { id } }
+    );
+    return res.status(204).json(nomination);
+  } catch (error) {
+    if (error) {
+      console.log('400 Update Bad Reuest', error);
+      return res.status(400).json({ error: error.message });
+    }
+    return res.status(500).json({ error: error.message });
```

Comment:
I don't think this return won't ever get hit as the return in the `if` statement will break out of the catch. 

in that case you can just remove the wrapping if statement to something like: 

```
} catch (error) {
  return res.status(500).json({error: error.message})
}
```
I updated to 500 as it's a server error and not a bad request (which is what 400 dictates)